### PR TITLE
feat(desktop-app): add light theme

### LINKED ISSUE #238: Add a light theme for the App Interface and have a switch in the preferences.
Hey guys! Just discovered this and it looks amazing! 

Sadly, my eyes prevent me from being able to work against a dark backgrounds, so I was keen to see if there was a Light mode. The Switch Color Scheme button doesn't seem to do anything when I click it — should that toggle the whole app from light to dark?

Thanks and fantastic work!

### PR DESCRIPTION
**Pull request by jzabala**
There are a lot of changes on this PR. Sorry about that 😢

Thanks @manojVivek for providing the base color scheme for the light mode 💪🏽

Screen Shot 2020-08-30 at 3 07 16 PM
Screen Shot 2020-08-30 at 3 07 29 PM
Screen Shot 2020-08-30 at 3 07 51 PM

Closes #238

### CODE REVIEW

**Comment by manojVivek** [comment]
@jzabala This is awesome! 🎉 👏 

The scrollbars got missed?

**Comment by jzabala** [comment]
> @jzabala This is awesome! 🎉 👏
> 
> The scrollbars got missed?

I didn't touch them 😅 .

Maybe a grey would be a better color.

**Comment by jzabala** [comment]
@manojVivek done. Better scrollbars.

Screen Shot 2020-08-31 at 9 23 46 PM

File: desktop-app/app/components/AddressInput.js [hunk 1]
```diff
@@ -312,4 +297,60 @@ class AddressBar extends React.Component<Props> {
...
+    ...commonStyles(theme),
+    addressBarContainer: {
+      display: 'flex',
+      position: 'relative',
+      alignItems: 'center',
+      height: '20px',
+      width: '100%',
+      padding: '14px 10px',
+      borderRadius: '20px',
+      backgroundColor: mode({light: theme.palette.grey['300'], dark: 'unset'}),
```
Comment (manojVivek) [inline desktop-app/app/components/AddressInput.js:312]: Is it possible to standardize these greys with a name? 
So that it can be used consistently?

**Comment by manojVivek** [comment]
> done. Better scrollbars.

Looks good now.

We might have to tweak the greys a bit I guess. But that should be easier, we can do it once we merge this in. 
There are a few conflicting files, we can merge this once you resolve those. 

@jzabala This is a big piece of work, great job pulling it together nicely! 👏 👏

File: desktop-app/app/components/AddressInput.js (hunk 1 above)
Comment (jzabala) [reply desktop-app/app/components/AddressInput.js:312]: The grey colors come by default in the mui theme: https://material-ui.com/customization/default-theme. 

I think we could standardize a couple if we agree to only use those but if we will use the whole range of grey colors in mui i would prefer to keep the number. It is hard to invent colors' names and projects usually end with: grey, lightGrey, lighterThanLightGrey, .... 😂

Comment (manojVivek) [reply desktop-app/app/components/AddressInput.js:312]: Yeah right. 😄 

Let's go with this and revisit it later when we have some clarity.

**Comment by manojVivek** [comment]
@jzabala Just the conflicts otherwise can be merged.

**Comment by jzabala** [comment]
> @jzabala Just the conflicts otherwise can be merged.

Ready @manojVivek . Conflicts fixed 💪🏽

**Pull request merged**